app/driver/onewire.c

```diff
@@ -180,7 +180,6 @@ void onewire_write(uint8_t pin, uint8_t v, uint8_t power /* = 0 */) {
   if ( !power) {
```

Comment:
Where can I read the sdk source like gpio_output_set? I am not sure whether the interruption is masked in some funcs